.env.example

```
@@ -21,6 +21,15 @@
 # Copy this file to .env and update with your values.
 # ============================================================================
 
+# -----------------
+# Web Interface
+# -----------------
+WEB_PORT=50110
+WEB_INTERNAL_PORT=3000
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The WEB_INTERNAL_PORT key should go before the WEB_PORT key

(UnorderedKey)